policyengine_us/variables/gov/states/ny/nyserda/drive_clean/nyserda_drive_clean_rebate.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+class ny_drive_clean_rebate(Variable):
+    value_type = float
+    entity = Household
+    label = "New York Drive Clean Rebate"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.nyserda.ny.gov/-/media/Project/Nyserda/Files/Programs/Drive-Clean-NY/implementation-manual.pdf#page=8"
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, parameters, period):
+        p = parameters(period).parameters.gov.states.ny.nyserda.drive_clean
+        # Calculate vehicle all-electric range
+        vehicle_range = tax_unit("nyserda_drive_clean_vehicle_electric_range", period)
+        # Calculate vehicle price
+        vehicle_cost = tax_unit("nyserda_drive_clean_vehicle_cost", period)
```

Comment:
```suggestion
        vehicle_cost = tax_unit("ny_drive_clean_vehicle_cost", period)
```

Reply:
if we change the class name, we will need to change the inputs in other vars as well as in the tests